tools/iptb-plugins/filecoin/local/localfilecoin.go

```diff
@@ -53,6 +56,9 @@ var (
 
 	// AttrLogJSON is the key used to set the node to output json logs
 	AttrLogJSON = "logJSON"
+
+	// AttrSectorsPath is the key used to set the sectors path
+	AttrSectorsPath = "sectorsPath"
```

Comment:
cc @travisperson there are a few ways to pass in sectorsPath.  Using an attribute seemed like the cleanest way to give iptb nodes a default of sector storage under the repo path.  

We could alternatively require all iptb users that want this config pass in the `--sectorsdir` flag upon init / enforce this option as part of fast environment setup.

Reply:
And of course we could always write to the config before calling `mining start` as part of the FAST series that sets up a mining node.

Reply:
I think this is good, but it does make it impossible to set the sector dir via the init flag I believe because it overrides it. Does the value default to empty in the config? If so, we could check the value first to see if it was set. I think this is fine though if we run into that issue we can fix it.

Reply:
Nice thinking.  Pushing this up now.